spanner/README.md

```diff
@@ -31,4 +31,87 @@ row, err := client.Single().ReadRow(ctx, "Users",
 if err != nil {
 	log.Fatal(err)
 }
+```
```

Comment:
@arpan14 @olavloite @rahul2393 @asthamohta 
Can you please take a look at the golang documentation for session leaks and give your LGTM?